frontend/src/types/user.d.ts

```diff
@@ -1,14 +1,17 @@
-import { Status, Role } from "./enums"
+import { Status, Role } from "./enums";
 
 type User = {
-    id: str,
-    image?: str,
-    name: str,
-    email: str,
-    password: str, 
-    role: Role, // (admin, user)
-    status: Status, // (active, inactive)
-    createdOn: Date;
-}
+  id?: str; //this is optional because when we create user, we don't need to provide the user id
+  image?: str;
```

Comment:
Sorry, I didn't notice this in the previous PRs and changes, but `str` is not a `string` type. In Typescript, the string type should be `string`. Can you help update this, thanks!